### PR TITLE
Add task file to enable service

### PR DESCRIPTION
**Pull request by nre-ableton**
This PR fixes an issue where the output of `launchctl print-disabled` differed between pre/post-Ventura macOS, and also introduces a new `enable-service.yml` task file to provide parity with `disable-service.yml`.

Note that `disable-service.yml` has also been adapted to only disable the service and not remove it.

### CODE REVIEW

**Review by mst-ableton — APPROVED** [review]
lgtm. Technically it's "Ventura-and-post-Ventura" but it would be a lot of work to adjust and I think the meaning is pretty clear to see in the code.

**Comment by ala-ableton** [comment]
@nre-ableton I'm interested in reviewing this PR. Thanks!

File: tasks/debian/enable-service-systemd.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (ala-ableton) [inline tasks/debian/enable-service-systemd.yml:1]: Typo in 92ab683b0b7ea67c904998ea9ffcaac13a06c7c3's commit message: compliment -> complement.

File: tasks/debian/enable-service-systemd.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- name: Enable systemd service
+  become: true
+  ansible.builtin.systemd:
+    name: "jenkins-swarm-client"
+    enabled: true
+    daemon_reload: true
+    state: reloaded
```
Comment (ala-ableton) [inline tasks/debian/enable-service-systemd.yml:8]: Why not use `state: started` here?

File: tasks/debian/enable-service-sysvinit.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Enable SysVinit service
+  become: true
+  ansible.builtin.sysvinit:
+    name: "jenkins-swarm-client"
+    enabled: true
+    state: reloaded
```
Comment (ala-ableton) [inline tasks/debian/enable-service-sysvinit.yml:7]: Why not use `state: started` here?

File: tasks/debian/enable-service-systemd.yml (hunk 2 above)
Comment (nre-ableton) [reply tasks/debian/enable-service-systemd.yml]: Fixed in 68d94a7.

**Review by ala-ableton — APPROVED** [review]

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**